src/main/java/org/scalameter/annotations/benchmark.java

```diff
@@ -0,0 +1,31 @@
+package org.scalameter.annotations;
```

Comment:
Yes, by "expose" I meant - make type aliases in the `org.scalameter.api` object for these annotations.

Didn't try, but that should help scalac resolve those annotations.